src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double areaCalculator();
```

Comment:
```suggestion
    double calculateArea();
```

названия методов должны начинаться с глагола